README.md

```diff
@@ -1,5 +1,43 @@
-# Cloud gaming demo for Anbox Cloud
+# Overview
 
-This demo provides an example of how to build a cloud gaming service with Anbox Cloud. It
-demonstrates interaction with the backend services Anbox Cloud provides and how streaming
-can be easily integrated.
+This project demonstrates a simple cloud gaming service on top of [Anbox Cloud](https://anbox-cloud.io)
+allowing users to play Android based games right from their web browser on any device.
+
+The demo is build of two components:
+
+* A web UI based on[Flutter](https://flutter.dev/)
+* A backend service written in [Python](https://www.python.org/)
+
+The backend service will connect to the Anbox Cloud services and retrieve a list of available applications
+and also take care of launching actual streaming session on demand.
+
+Inside the web UI the user can select from the list of available games and starting playing each of them.
+
+# Quickstart
+
+Running the demo yourself is as simple as running through the following steps:
+
+1. Install the Anbox Cloud Appliance. You can find more details [here](https://anbox-cloud.io/docs/tut/installing-appliance)
+2. SSH to your appliance instance, clone the demo repository and run the installation script:
+```
```

Comment:
I think this must be indented with three spaces to display correctly - but GitHub doesn't let me suggest that. ;)